src/ts/components/seekbar.ts

```diff
@@ -317,6 +318,26 @@ export class SeekBar extends Component<SeekBarConfig> {
     this.configureMarkers(player, uimanager);
   }
 
+  /**
+   * Update seekbar while a live stream with DVR window is paused.
+   * The playback position stays still and the position indicator visually moves towards the back.
+   */
+  private configureLivePausedTimeshiftUpdater(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager,
+                                              playbackPositionHandler: () => void): void {
+    // Regularly update the playback position while the timeout is active
+    const pausedTimeshiftUpdater = new Timeout(1000, playbackPositionHandler, true);
+
+    // Start updater when a live stream with timeshift window is paused
+    player.addEventHandler(player.EVENT.ON_PAUSED, () => {
+      if (player.isLive() && player.getMaxTimeShift() < 0) {
+        pausedTimeshiftUpdater.start();
+      }
+    });
+
+    // Stop updater when playback continues (no matter if the updater was started before)
+    player.addEventHandler(player.EVENT.ON_PLAY, () => pausedTimeshiftUpdater.clear());
```

Comment:
i think it won't hurt to add an additional `ON_PLAYING` handler

Reply:
It wouldn't hurt but I say it's unnecessary because
 * it's the player's job to fire the correct events, not the UI's job to fix player bugs
 * the UI is backwards compatible with all 7.x players and `ON_PLAYING` has only been added in 7.3

Reply:
I think there is no way to exit the paused state without an `ON_PLAY` event.